pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -692,7 +692,7 @@ type AzureSDConfig struct {
 	// The port to scrape metrics from. If using the public IP address, this must
 	// instead be specified in the relabeling rule.
 	// +optional
-	Port *int `json:"port"`
+	Port *int `json:"port,omitempty"`
```

Comment:
I think we can include minimum and maximum value for port number like did in EC2SDConfig
cc: @mviswanathsai

Reply:
thanks, fixed